CMakeLists.txt

```diff
@@ -65,6 +65,7 @@ endif()
 
 if(WIN32)
   set(APP_TARGET mudlet.exe)
+  set(MINGW_BASE_DIR C:\\Qt\\Tools\\mingw730_32)
```

Comment:
You'll need an extra test in here for the `USE_MAIN_BUILD_SYSTEM` CMake variable being true as that value `C:\\Qt\\Tools\\mingw730_32` is only going to be applicable for CI/CB builds and those using the original Windows development setup. If one is working the alternative pure MSYS2/Mingw-w64 setup like I use the value is likely to be something else.

If you look in the corresponding part of the QMake project file:
```
    } else:win32 {
        MINGW_BASE_DIR_TEST = $$(MINGW_BASE_DIR)
        contains( DEFINES, INCLUDE_MAIN_BUILD_SYSTEM ) {
            # For CI builds or users/developers using the setup-windows-sdk.ps1 method:
            isEmpty( MINGW_BASE_DIR_TEST ) {
                MINGW_BASE_DIR_TEST = "C:\\Qt\\Tools\\mingw730_32"
            }
            LIBS +=  \
                -L"$${MINGW_BASE_DIR_TEST}\\bin" \
                -llua51 \
                -lhunspell-1.6

            INCLUDEPATH += \
                 "C:\\Libraries\\boost_1_71_0" \
                 "$${MINGW_BASE_DIR_TEST}\\include" \
                 "$${MINGW_BASE_DIR_TEST}\\lib\\include"

        } else {
            # For users/developers building with MSYS2 on Windows:
            isEmpty( MINGW_BASE_DIR_TEST ) {
                error($$escape_expand("Build aborted as environmental variable MINGW_BASE_DIR not set to the root of \\n"\
                                      "the Mingw32 or Mingw64 part (depending on the number of bits in your desired\\n"\
                                      "application build) typically this is one of:\\n"\
                                      "'C:\msys32\mingw32' {32 Bit Mudlet built on a 32 Bit Host}\\n"\
                                      "'C:\msys64\mingw32' {32 Bit Mudlet built on a 64 Bit Host}\\n"\
                                      "'C:\msys64\mingw64' {64 Bit Mudlet built on a 64 Bit Host}\\n"))
            }
            LIBS +=  \
                -L$${MINGW_BASE_DIR_TEST}/bin \
                -llua5.1 \
                -llibhunspell-1.7

            INCLUDEPATH += \
                 $${MINGW_BASE_DIR_TEST}/include/lua5.1 \
                 $${MINGW_BASE_DIR_TEST}/include/pugixml
        }

        LIBS += \
            -lpcre-1 \
            -lzip \                 # for dlgPackageExporter
            -lz \                   # for ctelnet.cpp
            -lpugixml \
            -lWs2_32

        # Leave this unset - we do not need it on Windows:
        # LUA_DEFAULT_DIR =
    }
```
You'll see that we only force a value for `MINGW_BASE_DIR` if it hasn't been set already - and for the usage I make of it I ensure it is set to something like `C:\msys64\mingw64` normally or `C:\msys64\mingw32`if I am doing a 32-bit build...